docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,596 @@
+        existingSecret:
+          name: "camunda-credentials"
+        existingSecretKey: "identity-optimize-client-password"
+      core:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        tokenScope: "00000000-0000-0000-0000-000000000000/.default"
+        redirectUrl: "https://core.camunda.example.com"
+        existingSecret:
+          name: "camunda-credentials"
+        existingSecretKey: "identity-core-client-password"
+      console:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        wellKnown: https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/v2.0/.well-known/openid-configuration
+        audience: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        redirectUrl: "https://console.camunda.example.com"
+        existingSecret:
+          name: "camunda-credentials"
+        existingSecretKey: "identity-console-client-password"
+      webModeler:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        clientApiAudience: "00000000-0000-0000-0000-000000000000"
+        publicApiAudience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://modeler.camunda.example.com"
+      connectors:
+        existingSecret:
+          name: "camunda-credentials"
+        existingSecretKey: "identity-connectors-client-password"
+core:
+  contextPath: "/core"
+  ingress:
+    grpc:
+      enabled: true
+      className: nginx
+      host: "grpc-{{ .Values.global.ingress.host }}"
+      tls:
+        enabled: true
+        secretName: camunda-platform-core-grpc
+  retention:
+    enabled: true
+    minimumAge: 30d
+    policyName: core-record-retention-policy
+
+identity:
+  contextPath: "/identity"
+  firstUser:
+    existingSecret: "camunda-credentials"
+    existingSecretKey: "identity-user-password"
+  externalDatabase:
+    enabled: true
+    host: "external-postgres-host"
+    port: 5432
+    username: "identity_user"
+    database: "identity_db"
+    existingSecret: "identity-db-secret"
+    existingSecretPasswordKey: "database-password"
+
+optimize:
+  contextPath: "/optimize"
+
+connectors:
+  contextPath: "/connectors"
+
+webModeler:
+  enabled: true
+  contextPath: "/modeler"
+  restapi:
+    mail:
+      # This value is required, otherwise the restapi pod wouldn't start.
+      fromAddress: noreply@example.com
+  externalDatabase:
+    url: "jdbc:postgresql://external-postgres-host:5432/camunda_db"
+    user: "camunda_user"
+    existingSecret: "camunda-db-secret"
+    existingSecretPasswordKey: "database-password"
+
+elasticsearch:
+  enabled: false
+
+prometheusServiceMonitor:
+  enabled: true
+  labels:
+    release: kube-prometheus-stack
+```
+
+## Next Steps
+
+### Upgrade and Maintenance
+
+<!--INCOMPLETE-->
+
+Make sure secrets are not auto-generated on upgrade.
```

Comment:
We just need to mention that the secrets are not auto-generated on upgrade by default, and it's important not to override that.

Reply:
I added that points, thanks